### DISCUSSION TITLE
How to add host.docker.internal as extra host of Dokploy?

### DISCUSSION THREAD

**Discussion by Vincent-HD** [Dokploy/dokploy#727] [Q&A]
I need to reverse proxy something running on the host directly, so I need to have access to host.docker.internal from docker, but for this  I need to add something like we do in a compose file:
```
extra_hosts:
      - "host.docker.internal:host-gateway"
```
Can we achieve something similar for dokploy?

**Comment by Siumauricio** [comment]
Dokploy use native docker compose, so you shouldn't have any problems to add that property, we support all the properties except container_name prop

**Reply by Vincent-HD** [reply]
Hmm, sorry I wasn't clear, I added a custom router inside dynamic conf of traefik, that should points to the host's IP, for this I need to have the `host.docker.internal` that resolves correctly to the docker network gateway, so I needed to add this to the traefik container itself, because traefik needs to have access to be able to route the traffic to host ip from docker network.

I solved this by updating the service like this: `docker service update --host-add host.docker.internal:host-gateway dokploy-traefik`
